frontend/packages/ux-editor-v3/src/components/config/EditFormContainer.test.tsx

```diff
@@ -11,19 +11,23 @@ import {
   renderWithMockStore,
 } from '../../testing/mocks';
 import { useLayoutSchemaQuery } from '../../hooks/queries/useLayoutSchemaQuery';
-import { container1IdMock, externalLayoutsMock, layoutMock } from '../../testing/layoutMock';
-import { textMock } from '../../../../../testing/mocks/i18nMock';
+import {
+  container1IdMock,
+  externalLayoutsMock,
+  layoutMock,
+  layoutSet1NameMock,
+} from '../../testing/layoutMock';
```

Comment:
Should this be ⬇️?
```suggestion
} from '@studio/testing/layoutMock';
```

Reply:
No, it's not the same testing folder here. This one is inside the `ux-editor`